app/src/main/java/de/danoeh/antennapod/adapter/SubscriptionsAdapter.java

```diff
@@ -108,7 +109,7 @@ public View getView(int position, View convertView, ViewGroup parent) {
     public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
         Fragment fragment = FeedItemlistFragment.newInstance(getItemId(position));
         mainActivityRef.get().loadChildFragment(fragment);
-    }
+}
```

Comment:
Please do not touch unrelated files (this is probably a leftover from earlier experiments)

Reply:
Could you please undo the changes in this unrelated file? This makes the git history hard to read.